demo/components/date-picker-demo.tsx

```diff
@@ -13,11 +14,10 @@ export class DatePickerDemo extends React.Component<Partial<DatePickerProps>, Da
     public render() {
         return (
             <div>
-                <h3>Basic DatePicker</h3>
-                <span data-automation-id="CURRENT_DATE">
-                    {this.state.value.toDateString()}
-                </span>
+                <h2>Standard DatePicker</h2>
+                <span data-automation-id="CURRENT_DATE">{this.state.value.toDateString()}</span>
                 <DatePicker
+                    dataAutomationId="BASIC_DATEPICKER"
```

Comment:
you should be able to use `data-automation-id` after implementing `root` function